package.json

```diff
@@ -1,16 +1,14 @@
 {
   "name": "npm-packlist",
-  "version": "5.1.3",
+  "version": "5.0.0",
```

Comment:
this seems off?

Reply:
yeah it's not, that was a lazy rebase merge on my part